### DISCUSSION TITLE
Help! All trails failed when using NNI on Linux (Ubuntu 22.04.1 LTS)

### DISCUSSION THREAD

**Discussion by johnX2050** [microsoft/nni#5091] [Q&A] [answered]
**What I encountered** : I want to run my deep learning code with NNI on the Linux server.  Actually, it works well on local windows laptop. However, when I try to implement the same trail code, config(changed python to python3) and serach_space.json on Linux, all trails first are running but soon becoming failed. 

**What's the reason and how to fix it?**

**config**
experimentName: default
searchSpaceFile: search_space.json
trialCommand: python3 train.py 
trialConcurrency: 2
maxTrialNumber: 10              # Generate at most 10 trials.
maxExperimentDuration: 3h  # Stop generating trials after 1 hour.
#trialGpuNumber: 0
tuner:
  name: TPE
  classArgs:
    optimize_mode: minimize
trainingService:
  platform: local
  useActiveGpu: true

**search_space.json**
{
    "merge_ratio": {"_type": "choice", "_value": [0.2, 0.5, 0.8, 1, 1.5, 2]}
}

**Log:** 
![image](https://user-images.githubusercontent.com/95688790/187066158-c5a77e05-e2a0-40e8-ae62-86a1029816bd.png)
![image](https://user-images.githubusercontent.com/95688790/187066171-f7def0ea-99c8-4305-9e19-4ebbe23526d5.png)
![image](https://user-images.githubusercontent.com/95688790/187066194-7574a231-ffcf-45b3-8a40-545b2ed0da13.png)
![image](https://user-images.githubusercontent.com/95688790/187066214-0a647a09-5303-4e96-a7ec-c9f45416f381.png)

**Comment by johnX2050** [comment, marked as answer]
The solution I found is: if all trails fail at the same time, it probably be your environment is lack of some modules. Go to each trail log and find <stderr> to check which module you lost and reinstall them.

**Reply by XinY-Z** [reply]
How did you find the trail logs?
